sky/__init__.py

```diff
@@ -140,6 +140,7 @@ def set_proxy_env_var(proxy_var: str, urllib_var: Optional[str]):
 K8s = Kubernetes
 OCI = clouds.OCI
 Paperspace = clouds.Paperspace
+PrimeIntellect = clouds.PrimeIntellect
```

Comment:
Can we also add prime intellect to installation.rst? Here's an example: https://github.com/skypilot-org/skypilot/pull/4573/files#diff-d8aff290e858e583ffa7c44c47733ee1b132f3e229a2de8505d0dbb37f7d4729

Reply:
added in commit 198fb0b6d90134764c1ca0b37c2fe7661429de1c